packages/hadron-build/lib/download-center.js

```diff
@@ -73,9 +73,13 @@ let maybeUploadManifest = (CONFIG) => {
     channel: CONFIG.channel,
     platform: [
       {
-        name: 'OS X 10.10+ 64-bit',
+        name: 'MacOS 10.10+ 64-bit',
         download_link: `${prefix}/${CONFIG.id}-${CONFIG.version}-darwin-x64.dmg`
       },
+      {
+        name: 'MacOS 11.0+ arm64 (Apple M1)',
+        download_link: `${prefix}/${CONFIG.id}-${CONFIG.version}-darwin-arm64.dmg`
+      },
```

Comment:
AFAIK this is not used. When I joined we were updating the file on S3 manually, downloading it replacing all the versions, and re-uploading it. 

We automated that process here: https://github.com/mongodb-js/compass/blob/main/packages/compass/release/download-center.js#L5

but it still just does a replacement.

Reply:
Hmm, this script seems to be used by hadron-build upload command that we still have in the compass scripts but I don't know if we ever use it. I'll take a closer look and nuke the whole thing if it's not used just to avoid any confusion in the future

Reply:
Okay, so the `upload` script is used in CI to upload to (at least) github, not sure what's up with the manifest update then